src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -1519,6 +1519,37 @@ by { refine C.ind _, exact (λ _, rfl) }
   (C : G.connected_component) (φ : G →g G') (ψ : G' →g G'') : (C.map φ).map ψ = C.map (ψ.comp φ) :=
 by { refine C.ind _, exact (λ _, rfl), }
 
+/-- The set of vertices in a connected component of a graph. -/
+def connected_component.supp (C : G.connected_component) :=
+  { v | G.connected_component_mk v = C }
```

Comment:
```suggestion
G.connected_component_mk ⁻¹' {C}
```
is another, possibly more convenient, way to put it.

Reply:
We were considering it before making the PR, but decided to stick with the more verbose notation.

Reply:
I prefer the original notation: I feel it's a bit more transparent; e.g. when `C := G.connected_component_mk v`, membership is clear `rfl` while with the other notation you have (well, at least I do) to think to convince yourself that it unfolds to a `rfl`.